dunstctl

```diff
@@ -29,8 +29,9 @@ show_help() {
 	                                    notification with given ID.
 	  history-rm [ID]                   Remove the notification from
 	                                    history with given ID.
-	  is-paused                         Check if dunst is running or paused
-	  set-paused [true|false|toggle]    Set the pause status
+	  is-paused                         Check if pause level is > 0
+	  get-paused-level    Get the current pause level
```

Comment:
You removed `set-paused` here. I would keep `set-paused` as an easier option for pausing notifications for people who don't use pause levels. So I would not deprecate `is-paused` and `set-paused`.